src/convertToObject.js

```diff
@@ -6,7 +6,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  return sourceString.split(';').reduce((finalStyle, part) => {
+    if (part.trim()) {
+      const [command, value] = part.split(':');
+
+      finalStyle[command.trim()] = value.trim();
```

Comment:
[CODE KNOWLEDGE] - Be cautious when using `split(':')` as it may not work correctly if a CSS property value contains a colon. Although handling multiple colons is not required for this task, it would be a good practice to consider this case for a more robust function. If you decide to handle it, you could use a regex pattern or split at the first colon occurrence.